CONTRIBUTING.md

```diff
@@ -18,6 +18,8 @@
 
 Contributions are always welcome, no matter how large or small! Before contributing, please read the [code of conduct](https://github.com/babel/babel/blob/master/CODE_OF_CONDUCT.md).
 
+If you want an already configured online IDE to contribute to Babel, you can use Gitpod! [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/babel/babel)
```

Comment:
Not sure if it worths a badge. I think a text link is sufficient here.

Reply:
Agreed

Reply:
```suggestion
If you want an already configured online IDE to contribute to Babel, you can use [Gitpod](https://gitpod.io/#https://github.com/babel/babel)!
```

Reply:
@hzoo I used the badge in the readme because we already have the other badges.